Taskfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if [[ -z "${HISTORIC_TAG_DATA_PATH}" ]]; then
+    echo "[WARNING] You need to define HISTORIC_TAG_DATA_PATH, that will contain historic tag data. Exiting."
+    exit 1
+fi
+
+function build {
+    docker build -t daccs-image-updater:0.1 .
+}
+
+function run {
+    if [[ -z "${GITHUB_USER}" ]]; then
+        echo "[ERROR] Missing GITHUB_USER. Exiting."
+        exit 1
+    fi
+
+    if [[ -z "${GITHUB_PASSWORD}" ]]; then
```

Comment:
I guess `GITHUB_PASSWORD` will be used by `hub` tool, does the tool support token instead?  I think Github does have auth token.  More secure than using clear password.  Can revoke easily if needed.  Can have separate token for each "app" so can trace leak.